tests/CMakeLists.txt

```
@@ -686,6 +686,7 @@ list(APPEND ALL_2D_FILES ${FILES_2D} ${SCAD_DXF_FILES} ${SCAD_SVG_FILES})
 #
 # Add experimental features tests
 #
+experimental_tests(examples_roof)
```

The test output changed because the experimental flag was not applied. It needs the full name of the test case which is `<test-type>-<test-filename>`.
```suggestion
experimental_tests(dumptest-examples_roof)
```

---

I made the change you suggest, but this did not change the test output...

---

Did you rerun the top-level make to process the cmake file update and `TEST_GENERATE=1 ctest -R roof`?

---

I was builing in a separate folder (`build`). I did re-run cmake,make stuff in the test folder... Now I did a fresh cmake,make routine, building in the main folder, and it looks like it helped.